Cleanup changelog references in Introduction.md

stwrt opened the pull request:
#### 21704b5f7c44584a831986498dbcc8e0f85affd7
<pre>
Cleanup changelog references in Introduction.md
<a href="https://bugs.webkit.org/show_bug.cgi?id=242433">https://bugs.webkit.org/show_bug.cgi?id=242433</a>

Reviewed by Jonathan Bedard and Ryosuke Niwa.

Update Introduction.md to remove references to changelogs, and
enhance the &quot;commit messages&quot; section.

* Introduction.md:

Canonical link: <a href="https://commits.webkit.org/252241@main">https://commits.webkit.org/252241@main</a>
</pre>

JonWBedard commented on Introduction.md:
```diff
@@ -299,34 +299,10 @@ Running `git-webkit setup` configures your git hooks to properly generate commit
 Don’t worry about the “Reviewed by NOBODY (OOPS!)” line, the person landing your patch will fill this in.
-There is one ChangeLog per top-level directory.
 
-A typical change log entry before being submitted to [bugs.webkit.org](https://bugs.webkit.org/) looks like this:
```
Seems like we should provide a commit message example here.

stwrt replied:
Sure, I can add that

rniwa commented on Introduction.md:
```diff
@@ -296,37 +296,46 @@ It’s in your interest to wait for the Early Warning System to fully build and
+own navigation object and that the _isUserInitiated flag is correct for this
+navigation object.
+
+* Source/WebKit/UIProcess/API/Cocoa/WKNavigation.mm:
```
We want to use a commit message with per-function comments.

stwrt replied:
Found a commit with a couple of per-function comments.

rniwa commented on Introduction.md:
```diff
@@ -296,37 +296,47 @@ It’s in your interest to wait for the Early Warning System to fully build and
-There is one ChangeLog per top-level directory.
-
-A typical change log entry before being submitted to [bugs.webkit.org](https://bugs.webkit.org/) looks like this:
+The first line shall contain a short description of the commit message (this should be the same as the Summary field in Bugzilla). On the next line, enter the Bugzilla URL. Below the "Reviewed by" line, enter a detailed description of your changes. 
```
Can we insert more hard line breaks? This is a really long line.

stwrt replied:
Sure 😀

stwrt replied:
Added some notes about commenting here too.

stwrt committed:
Cleanup changelog references in Introduction.md
https://bugs.webkit.org/show_bug.cgi?id=242433

Reviewed by Jonathan Bedard and Ryosuke Niwa.

Update Introduction.md to remove references to changelogs, and
enhance the "commit messages" section.

* Introduction.md:

Canonical link: https://commits.webkit.org/252241@main
